code/src/index.css

```diff
@@ -1,13 +1,185 @@
 body {
   margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
-    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
-    sans-serif;
+  font-family: 'Montserrat', sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  background-color: black;
 }
 
 code {
-  font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
+  font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New',
     monospace;
 }
+
+.movie-wrapper {
+  display: grid;
+  grid-template-columns: repeat(1, 1fr);
+  grid-gap: 0;
+  width: 100%;
+}
+
+.movie-card .movie-text {
+  display: block;
+}
+
+.movie-card {
+  width: 100%;
+  position: relative;
+}
+
+.movie-image {
+  width: 100%;
+}
+
+.movie-text {
+  position: absolute;
+  bottom: 10px;
+  left: 20px;
+  z-index: 2;
+  display: none;
```

Comment:
And here you set the same class `.movie-text` as `display: none` so it's either block or none, depending on what you want.